README.md

```diff
@@ -348,6 +348,16 @@ OPTIONS
   --listr-renderer=default|silent|verbose  [default: default] Listr renderer
 ```
 
+To obtain a user access token from the indentity provider (eg, Keycloak or Red Hat Single Sign-On), using `oc` or `kubectl` with `curl` and `jq`:
+
+```
+KEYCLOAK_URL=$(kubectl get route/keycloak -n ${cheNamespace} -o jsonpath='{.spec.host}')
+USER_ACCESS_TOKEN=$(curl -X POST $KEYCLOAK_URL/auth/realms/che/protocol/openid-connect/token \
```

Comment:
Wrong url is here.
It should be `http://$KEYCLOAK_URL/auth/realms/che/protocol/openid-connect/token`

Reply:
what about https and make curl trust self-signed certificates.

Reply:
Yeah, we should default to https. What's the curl URL for self-signed cert?

Reply:
I mean is there a way to use --insecure to avoid issues like
  "Unable to communicate securely with peer: 
  requested domain name does not match the server's certificate"
in case of a poorly cert/PKI setup